catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -220,13 +230,33 @@ private void parseValidators(Map<String, Object> root, Outer outer) {
         }
 
         Map<String, List<Outer.Validator>> validators = new HashMap<>();
-        for (Map.Entry<String, Object> entry : ((Map<String, Object>) root.get(VALIDATORS_PROPERTY)).entrySet()) {
-            String rejson = JsonFactory.create()
-                    .toJson(entry.getValue());
-            List<Outer.Validator> lv = JsonFactory.create()
-                    .readValue(rejson, List.class, Outer.Validator.class);
-            validators.put(entry.getKey(), lv);
-        }
+        ((Map<String, Object>) root.get(VALIDATORS_PROPERTY)).forEach((attributeName, value) -> {
+            if (value instanceof ValueList) {
+                ValueList argumentList = (ValueList) value;
+                List<Outer.Validator> validatorList = new ArrayList<>();
+
+                for (Object object : argumentList) {
+                    if (object instanceof LazyValueMap) {
+                        LazyValueMap lazyValueMap = (LazyValueMap) object;
+                        final String comp = (String) lazyValueMap.get(VALIDATOR_PROPERTY);
+                        String json = OBJECT_MAPPER.toJson(object);
+                        Outer.Validator validator;
+
+                        switch (comp) {
+                        case MATCH_ANY:
```

Comment:
I'd add an inline comment here to clarify why currently you have two identical branches (the future intent being that there will be more).